### ISSUE TITLE
Regression in 3.8.0: `Unexpected positional argument` for `Time#iso8601(Integer)`

### ISSUE THREAD

**Issue by myronmarston** [ruby/rbs#2205]
After upgrading to RBS 3.8.0 and running `steep check` on my project, I'm getting a new error:

```
elasticgraph-indexer_lambda/lib/elastic_graph/indexer_lambda/sqs_processor.rb:110:62: [error] Unexpected positional argument
│ Diagnostic ID: Ruby::UnexpectedPositionalArgument
│
└         Time.at(seconds, millis, :millisecond).getutc.iso8601(3)
```

This appears to be caused by 50a15f64e8267cf2ddaf5919447ac75d06f82dec as part of #2189. The definition in `stdlib/time/0/time.rbs` supported an `::Integer` argument but the new definition in `core/time.rbs` does not.  However, Ruby continues to support that argument:

```
$ ruby --version
ruby 3.4.1 (2024-12-25 revision 48d4efcb85) +PRISM [arm64-darwin24]
$ ruby -e "puts ::Time.now.iso8601(1)"
2024-12-26T10:35:51.9-08:00
```

For anyone else who comes across this, the workaround is pretty simple:

https://github.com/block/elasticgraph/pull/67

(But I'd still like to see this fixed in RBS itself!).

**Comment by ParadoxV5** [comment]
also `#xmlshema`
```shell
$ ruby -e 'puts RUBY_VERSION, Time.now.xmlschema(3)'
3.4.1
2024-12-26T11:43:18.001-07:00
```

**Comment by soutaro** [comment]
@myronmarston thank you for reporting the issue. I'll fix the issue.

I also found I can access the repository! 😀 It can be a good example to test regression before releases.

**Comment by myronmarston** [comment]
Thanks for the quick fix!

**Comment by soutaro** [comment]
Released [3.8.1](https://github.com/ruby/rbs/releases/tag/v3.8.1) with the fix. 🎉
